client/build.rs

```diff
@@ -27,32 +27,28 @@ fn main() {
         .expect("Failed to run `rustfmt` on smartcontract");
     assert!(fmt.success(), "Can't format smartcontract");
 
-    let instrumenting_coverage = if let Ok(flags) = env::var("RUSTFLAGS") {
```

Comment:
Note from [here](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts):
> Note that since Rust 1.55, RUSTFLAGS is removed from the environment; scripts should use CARGO_ENCODED_RUSTFLAGS instead.